libexec/jenv-add

```diff
@@ -4,6 +4,11 @@
 set -e
 [ -n "$JENV_DEBUG" ] && set -x
 
+if [ "$#" -eq 0 ]; then
+  jenv help add
```

Comment:
Looking at [other instances of this](https://github.com/search?q=repo%3Ajenv%2Fjenv%20exit&type=code) we should call `jenv-help` directly and write to stderr e.g.
```suggestion
  jenv-help --usage add >&2
```